Studio/src/Application/Visualization/ShapeWorksStudioApp.cpp

```diff
@@ -342,6 +342,11 @@ bool ShapeWorksStudioApp::on_action_save_project_as_triggered()
   if (filename.isEmpty()) {
     return false;
   }
+
+  if (!StringUtils::hasSuffix(filename.toStdString(), "xlsx")) {
```

Comment:
should be ".xlsx" (missing dot)

Reply:
Yes indeed.  That goes for my other usage of hasSuffix.